test/sequential/test-util-debug-monkeypatched.js

```diff
@@ -0,0 +1,92 @@
+// Copyright Joyent, Inc. and other Node contributors.
```

Comment:
should it be 
```
Copyright Node.js contributors. All rights reserved.
SPDX-License-Identifier: MIT
```
As per https://github.com/nodejs/node/pull/10599#issuecomment-277318535